bitnami/etcd/templates/statefulset.yaml

```
@@ -149,7 +149,7 @@ spec:
             - name: ETCD_DATA_DIR
               value: "/bitnami/etcd/data"
             - name: ETCD_LOG_LEVEL
-              value: {{ ternary "debug" "info" .Values.image.debug | quote }}
+              value: {{ .Values.logLevel | quote }}
```

My suggestion is to use the `debug` when `image.debug` is set to _true_ and `logLevel` otherwise
```suggestion
              value: {{ ternary "debug" .Values.logLevel .Values.image.debug | quote }}
```